### PR TITLE
Add Redirection Section

### PR DESCRIPTION
**Pull request by vdnj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,6 +137,11 @@ Especially
 
 The whole configuration can be found inside [Cozy-UI/I18n folder](https://github.com/cozy/cozy-ui/blob/master/react/I18n)
 
+## Redirections
+
+If you need to make a link between cozy's apps, you should always use AppLinker from Cozy-UI. It handles a lot of use cases (flagship app, mobile app...).
+Never rely on `window.location` in the cozy's codebase. We do a few things that can have repercussion on this value. Always use the generateWebLink method. To use it, you'll have to get the subdomainType. And to get the subdomainType, you'll need to read it from the client: const { subdomain: subDomainType } = client.getInstanceOptions()
```
Comment (trollepierre) [inline README.md:143]:
```suggestion
Never rely on `window.location` in the cozy's codebase. We do a few things that can have repercussion on this value. Always use the generateWebLink method. To use it, you'll have to get the subdomainType. And to get the subdomainType, you'll need to read it from the client: `const { subdomain: subDomainType } = client.getInstanceOptions()`
```

**Review by JF-Cozy — APPROVED** [review]
Attention il manque les "docs: " devant les titre de tes commits

**Review by Merkur39 — APPROVED** [review]

**Comment by trollepierre** [comment]
+ let's squash the commits (we expect to see one commit only)

File: README.md [hunk 2]
```diff
@@ -137,6 +137,11 @@ Especially
 
 The whole configuration can be found inside [Cozy-UI/I18n folder](https://github.com/cozy/cozy-ui/blob/master/react/I18n)
 
+## Redirections
+
+If you need to make a link between cozy's apps, you should always use AppLinker from Cozy-UI. It handles a lot of use cases (flagship app, mobile app...).
+Never rely on `window.location` in the cozy's codebase. We do a few things that can have repercussion on this value. Always use the generateWebLink method. To use it, you'll have to get the subdomainType. And to get the subdomainType, you'll need to read it from the client: `const { subdomain: subDomainType } = client.getInstanceOptions()`
```
Comment (Ldoppea) [inline README.md:143]: I don't think this rule is true for the entire cozy's codebase, maybe we should refer to only cozy-apps?

For example, our Flagship mobile app inject `window.location` related code into webviews to handle some specific navigation scenario. Maybe some cloudery pages or the cozy-stack login pages also use it.

```suggestion
Never rely on `window.location` in the cozy-apps' codebase. We do a few things that can have repercussion on this value. Always use the generateWebLink method. To use it, you'll have to get the subdomainType. And to get the subdomainType, you'll need to read it from the client: `const { subdomain: subDomainType } = client.getInstanceOptions()`
```

**Review by Ldoppea — APPROVED** [review]

**Review by Ldoppea — APPROVED** [review]

**Comment by Ldoppea** [comment]
I approved, but don't forget to squash commits before merging

**Pull request merged**